package.json

```
@@ -31,7 +31,11 @@
     "roll-browser": "node utils/roll_browser.js",
     "coverage": "node test/checkCoverage.js",
     "check-deps": "node utils/check_deps.js",
-    "build-android-driver": "./utils/build_android_driver.sh"
+    "build-android-driver": "./utils/build_android_driver.sh",
+    "build-driver-64": "./utils/build-driver/build-driver-64.sh",
+    "build-driver-32": "@powershell -NoProfile -ExecutionPolicy Unrestricted -Command ./utils/build-driver/build-driver-32.ps1",
```

Can we label this win32? It took me a bit to figure out that -64 runs on linux and -32 runs on windows.
```suggestion
    "build-driver-win32": "@powershell -NoProfile -ExecutionPolicy Unrestricted -Command ./utils/build-driver/build-driver-32.ps1",
```